docs/github/code_conventions.md

```diff
@@ -0,0 +1,34 @@
+# Code style
+
+- Line width: 120 characters.
+- Use absolute imports instead of relative imports.
```

Comment:
I took the freedom to add this here... Did you have any update on https://github.com/Crypto-TII/CryptographicEstimators/issues/163?

Reply:
While running 
`sage test.py` as indicated in `add_new_estimator.md` I get a `ModuleNotFoundError:` error